CHANGELOG.md

```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
```

Comment:
When 2.0.0 is released will the changelog for the Pilot and the Release be combined? As a developer upgrading to 2.0.0 from 1.X I would expect all the changes to be in one place.

Reply:
I believe this should be the case, I will talk to Brandon Hawkins about this.